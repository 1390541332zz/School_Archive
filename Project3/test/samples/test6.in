File: test/maps/blacksburg_larger.asc
Least Elevation 0