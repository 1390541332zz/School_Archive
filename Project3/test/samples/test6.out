Loading: test/maps/blacksburg_larger.asc
Number of rows: 15
Number of columns: 20
	         0    1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16   17   18   19
	    0  586  689  813  801  605  659  585  522  545  554  547  542  542  640  688  740  750  780  956  961
	    1  866  916  727  601  516  516  565  602  561  538  571  574  546  567  625  594  662  680  704  800
	    2  845  687  558  534  628  692  833  995  857  600  594  564  540  551  543  539  567  600  621  647
	    3  580  553  565  626  833  918  977  938  729  617  590  555  595  596  579  545  538  562  573  579
	    4  653  743  859  932 1022  970  771  669  635  617  594  556  586  601  625  582  541  640  732  769
	    5  992 1015 1041 1093  859  734  677  641  604  580  555  559  578  632  633  645  625  602  630  674
	    6  941 1062 1096  996  879  750  689  648  647  623  586  582  600  631  690  652  594  646  702  699
	    7 1045 1006  926  917  997 1047  894  752  618  603  560  584  638  706  728  687  575  587  658  668
	    8  866  794  784  823  825  756  709  658  603  575  607  707  751  679  651  646  570  589  587  575
	    9  772  845  797  653  622  611  637  699  704  634  694  687  708  686  614  571  541  539  549  526
	   10  764  709  659  625  693  827  829  716  629  678  694  672  626  587  566  568  562  550  553  558
	   11  648  659  642  747  845  709  628  683  748  680  624  602  599  586  578  582  596  589  582  559
	   12  671  731  802  732  657  714  758  710  643  618  600  607  613  608  609  605  601  600  580  557
	   13  836  769  666  742  774  701  650  629  626  619  624  632  632  620  610  601  585  583  555  566
	   14  697  699  777  707  663  650  644  646  640  644  654  650  633  628  618  604  601  566  573  597
	
Command: Least Elevation
	Height change:  -7
	Steps:          21
	Average change: -0.333333
	Starting row:   0
Number of rows: 15
Number of columns: 20
	     0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19
	 0   >  \  -  >  >  >  \  -  -  >  >  >  v  -  -  -  -  -  -  -
	 1   -  -  /  -  -  -  -  >  /  -  -  -  \  -  -  -  -  -  -  -
	 2   -  -  -  -  -  -  -  -  -  -  -  -  -  >  \  -  -  -  -  -
	 3   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  \  >  >  >  X
	 4   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  ^  -  -  -
	 5   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 6   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 7   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 8   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 9   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	10   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	11   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	12   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	13   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	14   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	
